File: Reservamos/app/src/main/java/com/reservamos/SplashActivity.java
package com.reservamos;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

/**
 * Created by pablo on 3/1/2018.
 */

public class SplashActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        Intent intent = new Intent(this, MainActivity.class);
        startActivity(intent);
        finish();
    }
}

File: ReservamosKotlin/app/src/main/java/com/pabloeferreyra/reservamos/SplashActivity.kt
package com.pabloeferreyra.reservamos

import android.content.Intent;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.WindowManager;
/**
 * Created by pablo on 23/12/2017.
 */
 open class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val intent = Intent (
                this,
                MainActivity ::class.java
        )
        startActivity(intent)
        finish()
    }
}
